ws-renderer/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": [["next/babel"]],
+  "plugins": ["@emotion/babel-plugin","@babel/plugin-proposal-export-default-from", "@loadable/babel-plugin"]
```

Comment:
the loadable plugin here doesn't work and should be removed if an extended spike is attempted as per the PR description